packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,81 @@
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/datetime'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-datetime--basic'
+typescript: ./src/DateTime.tsx,.RelativeDateTime.tsx
+---
+
+In this package, there are two components (`DateTime` and `RelativeDateTime`) that will return a date following [Forma 36 guidelines](https://f36.contentful.com/guidelines/date-and-time/).
+
+## DateTime
+
+The DateTime component will format any date to a human friendly format and wrap it in a `<time>` tag.
+It allows four different forms of display that can be controlled by changing the `format` prop:
+
+| `format` option | Result                 |
+| --------------- | ---------------------- |
+| `full`          | 12 Aug 2020 at 8:00 AM |
+| `day`           | 12 Aug 2020            |
+| `time`          | 8:00 AM                |
+| `weekday`       | Wed, 12 Aug            |
+
+### Examples
+
+```tsx
+import { DateTime } from '@contentful/f36-datetime';
+
+<DateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<DateTime date={new Date()} />; // passing a JS Date
+<DateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
+```
+
+All the usages above will result in 'Tue, 17 Aug 2021 at 3:45 PM'
+
+## RelativeDateTime
+
+Provides a relative date string for resolutions ranging from seconds up through years. Should be used when absolute date and time is unnecessarily specific.
+
+### Examples
+
+If we consider today as August 18th of 2021 and we use pass the following values to the Component
+
+```tsx
+import { RelativeDateTime } from '@contentful/f36-datetime';
+
+<RelativeDateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<RelativeDateTime date={new Date()} />; // passing a JS Date
+<RelativeDateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
+```
+
+The component will display `a day ago`
+
+### Passing a `baseDate`
+
+Normally, the component compare your `date` with today’s date. But maybe you need to render a relative date
+that is compared with some other day that is not today. You can do that by using `baseDate`:
+
+```tsx
+import { RelativeDateTime } from '@contentful/f36-datetime';
+
+<RelativeDateTime baseDate="2021-08-17T15:45" date="2021-08-18T15:45" />;
+```
+
+The component will display `in a day`
+
+### Using `isRelativeToCurrentWeek`
+
+Still using the last example. If you only need a relative date if the given date is within the current week,
+that is possible by using the boolean `isRelativeToCurrentWeek`
+
+```tsx
+import { RelativeDateTime } from '@contentful/f36-datetime';
+
+<RelativeDateTime date="2021-08-17T15:45" isRelativeToCurrentWeek />; // It will display "Yesterday at 3:45 PM"
+
+<RelativeDateTime date="2021-08-18T10:45" isRelativeToCurrentWeek />; // It will display "5 hours ago"
+
+<RelativeDateTime date="2021-08-18T15:45" isRelativeToCurrentWeek />; // It will display "Tomorrow at 3:45 PM"
+```
```

Comment:
PropTypes table is needed to be added explicitly.

Reply:
Oh I forgot it again! 🤦